apps/web/src/services/cost-explorer/CostExplorerLNB.vue

```diff
@@ -163,9 +163,13 @@ const filterFavoriteItems = (menuItems: LNBItem[] = []): LNBItem[] => {
     return menuItems.filter((menu) => (menu.id && state.favoriteItemMap[menu.id]) || menu.type !== MENU_ITEM_TYPE.ITEM);
 };
 
-const getCurrenctCurrency = (dataSourceKey: string): string => dataSourceState.dataSourceMap[dataSourceKey].data.plugin_info.metadata.currency;
-
-const getCurrencySymbol = (currency: string): string => CURRENCY_SYMBOL[currency];
+const getCurrentCurrencySet = (dataSourceKey: string): string => {
+    const defaultCurrencySet = `${CURRENCY_SYMBOL.USD}${CURRENCY.USD}`;
+    const currentCurrency = dataSourceState.dataSourceMap[dataSourceKey].data.plugin_info.metadata.currency;
```

Comment:
question: no need to set optional mark in here?

Reply:
I think it is necessary. Thanks!